website/static/js/logTextParser.js

```diff
@@ -25,6 +25,12 @@ function ravenMessage (message, logObject) {
     }
 }
 
+function toRelativeUrl(url) {
+    var parser = document.createElement('a');
```

Comment:
Go ahead and link to that stack overflow question in a comment in your function.